src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -1298,7 +1300,7 @@ def test_sql_db_long_term_retention(
 
 
 class SqlServerDbGeoRestoreScenarioTest(ScenarioTest):
-    @record_only()
+    @live_only() # Adding the live_only label after discussing with test owner rebeccaxu as the test was initially recorded on existing fixed resources.
```

Comment:
Can you/rebeccaxu update the test to not use fixed resources?

Reply:
Not sure if not using fixed resources is possible for this scenario. See dev note on next line: "using fixed resources because of long time preperation for geo-redundant backup"